blog.html

```diff
@@ -3,13 +3,9 @@
 ---
 <h2>Posts</h2>
 
-<ul class="post-list">
-    {% for post in site.categories.blog %}
-    <li>
-        <h3>
-            <span class="post-meta post-first-col">{{ post.date | date: "%b %-d, %Y" }}</span> <a class="post-link"
-                href="{{ post.url | prepend: site.baseurl }}">{{ post.title }}</a>
-        </h3>
-    </li>
-    {% endfor %}
-</ul>
+{% for post in site.categories.blog %}
```

Comment:
Let's keep the markup semantic -- it is a list, so I think we should use `ul` & `li` tags.

Reply:
Done.